R/compute-npc.r

```diff
@@ -17,7 +17,6 @@
 #'
 #' @keywords internal
 #'
-#' @export
```

Comment:
@aphalo Just a quick explanation here: I saw the `@keywords internal` above, so it looked to me that we should not have this function be part of the user interface. Also the description / functionality pointed towards that. Please let us know if it should still be exported, thank you

Reply:
@danielinteractive These functions are imported by another of my own packages, 'ggpmisc'. I have marked them as internal probably before I split the original 'ggpmisc' into 'ggpp' and 'ggpmisc'. I think the code is stable enough so that `@keywords internal` can be deleted if needed.

Reply:
ah ok, makes sense. @sob2021 can you keep the `@export` and remove the `@keywords internal`, and add unit tests for these functions please? thanks :-)

Reply:
@sob2021 @danielinteractive I will remove `@keywords internal` and at the same check that the documentation is not too skinny.

Reply:
Hi @aphalo 
I've removed this on this branch and reinstated the @export on this branch. Hope this is OK.

Reply:
@sob2021 @danielinteractive The edit is now done in Master and pushed to GitHub.

Reply:
thx @aphalo - @sob2021 can you give me developer access to your forked repo? then I could push this update for you.